scripts/flu_hosp_prod.R

```diff
@@ -264,7 +264,25 @@ rlang::list2(
           select(geo_value, source, target_end_date = time_value, value) %>%
           filter(target_end_date > truth_data_date, geo_value %nin% insufficient_data_geos) %>%
           mutate(target_end_date = target_end_date + 6)
-        truth_data <- nhsn_latest_data %>%
+        forecast_date <- as.Date(forecast_generation_date)
```

Comment:
Hm thought we separated out forecast_date from forecast_generation_date? Let's make forecast_date a variable up top? Mostly I want the pipeline runable on Wednesday or Thursday, in case delays happen again.

Reply:
yeah we're a little inconsistent about the use of the two, thanks to the variable mangling. I guess we should just overwrite `forecast_generation_date` in each target

Reply:
Yea, I feel like `forecast_generation_date` should 99% of the time just be Sys.Date(), but it's handy to have it named and adjustable (if for whatever reason someone wants to spoof making reports on another day).